nixos/modules/services/monitoring/nagios.nix

```diff
@@ -77,10 +84,13 @@ in
     (mkRemovedOptionModule [ "services" "nagios" "urlPath" ] "The urlPath option has been removed as it is hard coded to /nagios in the nagios package.")
   ];
 
+  meta.maintainers = with lib.maintainers; [ symphorien ];
+
   options = {
     services.nagios = {
       enable = mkOption {
```

Comment:
Is it not possible to use `mkEnableOption` here?

Reply:
Yeah that is nice to do.